go.mod

```diff
@@ -260,6 +262,7 @@ require (
 	github.com/open-telemetry/opentelemetry-collector-contrib/pkg/sampling v0.102.0 // indirect
 	github.com/open-telemetry/opentelemetry-collector-contrib/pkg/translator/azure v0.102.0 // indirect
 	github.com/open-telemetry/opentelemetry-collector-contrib/pkg/translator/prometheus v0.102.0 // indirect
+	github.com/open-telemetry/opentelemetry-collector-contrib/receiver/azuremonitorreceiver v0.98.0 // indirect
```

Comment:
Failing CI/CD action output:
```
One or more Go files are not tidied. Run 'make for-all CMD=make tidy' and push the changes.
```
This dep will need to be updated to latest, `v0.102.0` instead of `v0.98.0`.